dashboardsly/config.py

```diff
@@ -1,6 +1,37 @@
+import json
 import os
 
 
+def get_banner_links():
+    banner_links = [
+        {'href': 'https://google.com', 'text': 'Financials'},
```

Comment:
The links are unchanged from what's there now.  @chriddyp should we change to something other than Google?